src/CourseDashboard.cpp

```diff
@@ -51,34 +54,88 @@ void CourseDashboard::updateUser(User & user)
     std::cout << "Which data You want to edit? Give number: " << '\n';
     std::cin >> number;
     std::string name, nick, group, gitHub, firecode;
-    switch (number) {
-    case 1:
-        std::cout << "Give new value to data: ";
-        std::cin >> name;
-        user.setName(name);
-        break;
-    case 2:
-        std::cout << "Give new value to data: ";
-        std::cin >> nick;
-        user.setNick(nick);
-        break;
-    case 3:
-        std::cout << "Give new value to data: ";
-        std::cin >> group;
-        user.setGroup(group);
-        break;
-    case 4:
-        std::cout << "Give new value to data: ";
-        std::cin >> gitHub;
-        user.setGitHub(gitHub);
-        break;
-    case 5:
-        std::cout << "Give new value to data: ";
-        std::cin >> firecode;
-        user.setFirecode(firecode);
-        break;
-    default:
-        std::cout << "You gave wrong value.";
-        break;
+    switch (number)
+    {
+        case 1:
+            std::cout << "Give new value to data: ";
+            std::cin >> name;
+            user.setName(name);
+            break;
+        case 2:
+            std::cout << "Give new value to data: ";
+            std::cin >> nick;
+            user.setNick(nick);
+            break;
+        case 3:
+            std::cout << "Give new value to data: ";
+            std::cin >> group;
+            user.setGroup(group);
+            break;
+        case 4:
+            std::cout << "Give new value to data: ";
+            std::cin >> gitHub;
+            user.setGitHub(gitHub);
+            break;
+        case 5:
+            std::cout << "Give new value to data: ";
+            std::cin >> firecode;
+            user.setFirecode(firecode);
+            break;
+        default:
+            std::cout << "You gave wrong value.";
+            break;
+    }
+}
+
+void CourseDashboard::saveToFile()
+{
+    std::fstream file_DB("baza danych.txt", file_DB.in | file_DB.out);
+    if(!file_DB.is_open())
+    {
+        std::cout << "error" << std::endl;
+    }
+    else
+    {    
+        for (auto & user : users_)
+        {
+            file_DB << user.getAllInfo();
+        }
+    }
+    file_DB.close();
+}
+
+void CourseDashboard::loadFromFile()
+{
+    std::fstream file_DB("baza danych.txt", file_DB.in);
```

Comment:
maybe the file name should be a parameter to this function? You can use ifstream and skip the in flag.